public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -28,16 +28,16 @@ export interface ConnectedProps {
 export type Props = OwnProps & ConnectedProps;
 
 const PanelInspectorUnconnected = ({ panel, dashboard, plugin }: Props) => {
+  const location = useLocation();
+  const defaultTab = new URLSearchParams(location.search).get('inspectTab') as InspectTab;
   const [dataOptions, setDataOptions] = useState<GetDataOptions>({
-    withTransforms: false,
+    withTransforms: defaultTab === InspectTab.Error,
```

Comment:
previously this would mask any xform errors since they are not run

Reply:
yea, working on the new inspector for scenes and I looked at this code and its very confusing that the data options that live under the Data tab can affect the data passed to other tabs. (Not doing that for scene inspect drawer)